app/src/androidTest/java/br/com/catbag/giffluxsample/matchers/Matchers.java

```diff
@@ -0,0 +1,94 @@
+package br.com.catbag.giffluxsample.matchers;
+
+import android.graphics.drawable.ColorDrawable;
+import android.os.IBinder;
+import android.support.test.espresso.Root;
+import android.support.test.espresso.matcher.BoundedMatcher;
+import android.support.test.internal.util.Checks;
+import android.view.View;
+import android.view.ViewGroup;
+import android.view.ViewParent;
+import android.view.WindowManager;
+
+import org.hamcrest.Description;
+import org.hamcrest.Matcher;
+import org.hamcrest.TypeSafeMatcher;
+
+/**
+ * Created by niltonvasques on 10/14/16.
+ */
+
+public class Matchers {
+    private static Matcher<View> childAtPosition(
+            final Matcher<View> parentMatcher, final int position) {
+
+        return new TypeSafeMatcher<View>() {
+            @Override
+            public void describeTo(Description description) {
+                description.appendText("Child at position " + position + " in parent ");
+                parentMatcher.describeTo(description);
+            }
+
+            @Override
+            public boolean matchesSafely(View view) {
+                ViewParent parent = view.getParent();
+                return parent instanceof ViewGroup && parentMatcher.matches(parent)
+                        && view.equals(((ViewGroup) parent).getChildAt(position));
+            }
+        };
+    }
+
+    public static Matcher<View> withBGColor(final int color) {
+        Checks.checkNotNull(color);
+        return new BoundedMatcher<View, View>(View.class) {
+            @Override
+            public boolean matchesSafely(View warning) {
+                int currentColor = ((ColorDrawable)warning.getBackground()).getColor();
+                System.out.println("Current color: "+currentColor);
+                return color == currentColor;
+            }
+            @Override
+            public void describeTo(Description description) {
+                description.appendText("with background color: "+color);
+            }
+        };
+    }
+
+    public static Matcher<View> animationRunning() {
```

Comment:
I've wrote some matchers to help test the UI. This I wrote to test gif animation, but I've discovered that Glide doesn't uses animations to play gif. On this project, if we have some animations we will eventually need use this matcher.